More cleanup to minimize our fork, some of which was upstreamed

sparist opened the pull request.

nickeliot reviewed (approved):
I don't think the "/nq" flag in install_wrapper.cc (line 61) is necessary anymore.  It was added to force installs from the meta installer to be silent.     I have since discovered that the meta installer can be tagged with a "silent" option, achieving the same functionality.

sparist commented:
I added the /qn back to being the code in line with upstream, and because I think its removal caused interactive dialogs to appear during unit tests. If you think it should be removed and replaced with the silent option, maybe you could do that in a branch and MR for dcp/develop.

nickeliot commented:
Maybe I have it backwards.  I did notice the dialogs during the unit tests and didn't make the connection.  
What ever it was before I started hacking is the way it should be.
